data/testng/versions/677302c/version.yml

```diff
@@ -9,4 +9,5 @@ misuses:
 - 'dmmc-4'
 - 'grouminer-4'
 - 'grouminer-17'
+- 'mudetect-15'
```

Comment:
Where's this coming from?

Reply:
That's actually a good question. I am sorry. I reset the file to the commit 76edd65d831070b28d5f4b75e09dcf03961ee771 in commit c042a9d80c07808ee9e9f4477398b7c967866613.